src/Public/js/myradio.form.js

```diff
@@ -356,8 +356,14 @@ window.MyRadioForm = {
         if (typeof tinymce !== 'undefined') {
             tinymce.init(
                 {
-                    selector: "textarea.myury-form-tinymce",
-                    plugins: "anchor autolink charmap code contextmenu fullscreen hr image link lists media tabfocus table wordcount",
+                    selector: "textarea.myradio-form-tinymce",
+                    plugins: "anchor autolink charmap code contextmenu hr image link lists media tabfocus table wordcount",
+                    media_alt_source: false,
+                    menubar: false,
+                    toolbar: "undo redo | styleselect | bold italic underline | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image media charmap | code",
+                    invalid_elements: "script",
```

Comment:
I was thinking about this, and some styling might be wanted, so I've blocked colour font-size background attributes, I wasn't sure whether to just remove style entirely or not.